projects/igniteui-angular/src/lib/directives/date-time-editor/date-time-editor.directive.spec.ts

```diff
@@ -617,7 +617,7 @@ describe('IgxDateTimeEditor', () => {
                 const longTimeOptions = { hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: true };
                 result = ControlsFunction.formatDate(date, longTimeOptions);
                 const offset = date.getTimezoneOffset();
-                const tz = (offset > 0 ? '-' : '+') + (Math.abs(offset) / 60);
+                const tz = (offset > 0 ? '-' : '+') + (Math.abs(offset) % 60);
```

Comment:
Erm, neither the division or the remainder are correct on their own 😁
It should be <rounded division>:<remainder>